acquisitions/src/main/resources/thunderjet/mod-orders/features/open-orders-with-poLines.feature

```diff
@@ -0,0 +1,82 @@
+# For https://folio-org.atlassian.net/browse/MODORDERS-1226
+Feature: Open Orders with PoLines
+
+  Background:
+    * url baseUrl
+    * print karate.info.scenarioName
+
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    # Define reusable functions
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+
+  @Positive
+  Scenario: Open Orders with PoLines
+   # Generate unique UUIDs
+    * def fundId = call uuid
+    * def budgetId = call uuid
+    * def orderId = call uuid
+
+    * print '1. Prepare finances'
+    * configure headers = headersAdmin
+    * def v = call createFund { id: '#(fundId)' }
+    * def v = call createBudget { id: '#(budgetId)', fundId: '#(fundId)', allocated: 1000 }
```

Comment:
we can move to background https://github.com/folio-org/folio-integration-tests/blob/8c588ef9bf0563f484cd0659249e43605de7813c/acquisitions/src/main/resources/thunderjet/mod-orders/features/pieces-batch-update-status.feature#L29